pkg/controller/versions/strategy_v2_1.go

```diff
@@ -312,12 +311,8 @@ func (v *versionStrategyV2_1) Render(ctx context.Context, cr *common.ControllerR
 		return nil, fmt.Errorf("Unexpected error setting Status.AppliedSpec: %v", err)
 	}
 
-	// Read in global.yaml
-	values, err := chartutil.ReadValuesFile(path.Join(v.GetChartsDir(), "global.yaml"))
-	if err != nil {
-		return nil, fmt.Errorf("error reading global.yaml file")
-	}
-	values.MergeInto(spec.Istio.GetContent())
+	// global.yaml is removed and combined into values.yaml of each chart
```

Comment:
Didn't you add global.yaml back? Doesn't this code now need to be here again?

Reply:
Yes, I added it back because chart rendering was failing due to a lot of missing global values and it was hard to figure out which changes are from upstream and what is coming from maistra charts.

Thanks, I will revert it back.